samples/common/ethernet.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(golioth_eth, LOG_LEVEL_DBG);
+
+#include <samples/common/ethernet.h>
+#include <zephyr/net/net_if.h>
+
+void eth_connect(void)
+{
+	LOG_INF("Connecting to Ethernet");
+	struct net_if *iface;
+	iface = net_if_get_default();
+	net_dhcpv4_start(iface);
+}
```

Comment:
I don't think "Connecting to Ethernet" is a correct form. It is just starting DHCP. How about just putting `net_dhcpv4_start(net_if_get_default());` in samples instead of `eth_connect()`?.

Reply:
Good call, I'm removing this file and following your suggested pattern.